### ISSUE TITLE
Colour changing does not work on a light

### ISSUE THREAD

**Issue by theLMGN** [milo526/homebridge-tuya-web#390]
**Describe the bug**
The light selected in the Home app accurately reflects the colour of the light. Instead, trying to change the colour of the light simply sets it to a blue colour, and it is required to enter the Smart Life app to return it to the white colour.

**Steps To Reproduce**
1. Have a light bulb on your Tuya account
2. Change the colour
**Expected:** The colour of the light bulb changes to the colour selected in the Home app

**Reality:** The colour of the light bulb changes to blue

**Plugin Version**
 
 0.1.13
 **Homebridge Config**

 
 ```json

            "options": {
                "username": "xxxxxxxxxxx@gmail.com",
                "password": "xxxxxxxxxxx",
                "countryCode": "44",
                "platform": "smart_life",
                "pollingInterval": 10,
                "scene": false
            },
            "name": "TuyaWebPlatform",
            "platform": "TuyaWebPlatform"
        

 ```

 
 **Device Config**
 
```

 mobile@iPad ? /p/v/mobile ? python3 tuya.py Wed Jan  5 03:15:32 2022Getting credentials
Got credentials
{   'access_token': 'EUheu1606649xxxxxxxxxx',
    'expires_in': 864000,
    'refresh_token': 'EUheu1606649xxxxxxxxxx',
    'token_type': 'bearer'}
Getting devices
Got devices
{   'header': {'code': 'SUCCESS', 'payloadVersion': 1},
    'payload': {   'devices': [   {   'data': {   'brightness': '1000',
                                                  'color_mode': 'colour',
                                                  'color_temp': 15224,
                                                  'online': True,
                                                  'state': 'true'},
                                      'dev_type': 'light',
                                      'ha_type': 'light',
                                      'icon': 'https://images.tuyaeu.com/smart/solution/28002/37e69c2be34b7642_cover.png',
                                      'id': '474172802cf432e2d525',
                                      'name': 'Ligh'}],
                   'scenes': []}}

```
 
 **Logs**
```

 [1/5/2022, 3:13:21 AM] Homebridge v1.3.8 (HAP v0.9.7) (Homebridge 777C) is running on port 53308.

[1/5/2022, 3:13:21 AM] [TuyaWebPlatform] Existing Accessory found [Ligh] [474172802cf432e2d525] [3e98a54d-8dce-44cd-b3d2-b8e51a27467d]

[1/5/2022, 3:13:22 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Hue': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics. See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:22 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Saturation': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics. See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:22 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 300 exceeded maximum of 100. See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:41 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 300 exceeded maximum of 100. See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:41 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Saturation': characteristic value expected valid finite number and received "undefined" (undefined). See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:41 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Hue': characteristic value expected valid finite number and received "undefined" (undefined). See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:41 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 300 exceeded maximum of 100. See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:41 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Hue': characteristic value expected valid finite number and received "undefined" (undefined). See https://git.io/JtMGR for more info.

[1/5/2022, 3:13:41 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Saturation': characteristic value expected valid finite number and received "undefined" (undefined). See https://git.io/JtMGR for more info.

[1/5/2022, 3:14:32 AM] [@fratton/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 300 exceeded maximum of 100. See https://git.io/JtMGR for more info.

```

**Comment by theLMGN** [comment]
(I am still waiting on a response)

**Comment by milo526** [comment]
Are we talking about colors or color temperature.
I.e. red/green/purple or warm light (more yellowish) and cool lights (more blueish)

**Comment by theLMGN** [comment]
Both. If I change either the temperature or color it changes to white in the colour mode. The hue seems to update, but the saturation seems to always be 0 and the brightness is set to full, so it appears as white.

https://photos.google.com/share/AF1QipPVyOxm3kyaJW4OP4Gl-RT8W4aOp9ynk5Hw4gZFcQV6Uw8OAb2-3CfIt7oPUZCWMg/photo/AF1QipNnE0FsiPsUsOG_sunDWdV9qc88X-JBkHu2rR5T?key=emFqbEJwdTFhX3hheVMzenV4OUh4eFJNaHZFRkJR

**Comment by milo526** [comment]
This seems to be a duplicate of #387, will close in favour of that one.
